### ISSUE TITLE
Aria fields are tampered with by foundations ABIDE javascript

### ISSUE THREAD

**Issue by kevinmu17** [foundation/foundation-sites#12098]
I have a form with inputs with only aria-errormessage. Somehow foundation is duplicating this and pasting this on the same element as an aria-describedby. This is unwanted behaviour!

## What should happen?
Abide should leave things untouched :)

## What happens instead?
Duplicates aria-errormessage as an aria-describedby on the same elements AND elements wich completly lacks aria fields (hidden inputs) with the reference to the first input which holds the aria-errormessage

## Possible Solution
It's seems this bug snugged in since 6.6.3 within ABIDE, but somehow i can't reproduce this issue in a codepen. But I have 2 projects online which both have this same issue.

## Test Case and/or Steps to Reproduce (for bugs)
Test Case: https://codepen.io/kevinmu/pen/LYGxjeE

How to reproduce:
1. Start a project with the foundation CLI
2. Add a form from the foundation abide docs
3. Inspect all inputs, they should have all magically have an aria-describedby when not applied yourself

## Context
I'm not sure IF it affects me, but a client of mine reported she can't submit the form on a fairphone with firefox, so when investigating this I found this malfunction. Don't know if this is a related issue or something is just broke in Abide.

## Your Environment
- Foundation version(s) used: 6.6.3
- Browser(s) name and version(s): Chrome 83.0.4103.106 (Official Build) (64-bit)
- Device, Operating System and version: Imac OSX 10.13.6
- Link to your project: n/a

## Checklist
- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.

**Comment by DanielRuf** [comment]
> I'm not sure IF it affects me, but a client of mine reported she can't submit the form on a fairphone with firefox, so when investigating this I found this malfunction. Don't know if this is a related issue or something is just broke in Abide.

Highly not related as aria attributes are only relevant for assistive solutions and screenreaders.

> It's seems this bug snugged in since 6.6.3 within ABIDE, but somehow i can't reproduce this issue in a codepen. But I have 2 projects online which both have this same issue.

Then it is probably a bug in the project. You can export the codepen on the bottom right as zip and test locally.

As it can not be reproduced on codepen it can not be fixed:

![image](https://user-images.githubusercontent.com/827205/85131056-5dce4a00-b236-11ea-8d4a-17264b16f575.png)

I see no error there:

https://github.com/foundation/foundation-sites/blob/v6.6.3/js/foundation.abide.js#L295-L310

And I see no related changes there:
https://github.com/foundation/foundation-sites/commits/develop/js/foundation.abide.js

Please check if it happens with previous versions too and by removing scripts and disabling browser extensions to see what causes this.

**Comment by DanielRuf** [comment]
Also keep in mind to properly wrap fields.
